[Edge] Define a new profile "Edge Profile" (#1692)
cntt-n/CNTT: doc/ref_model/chapters/chapter02.md

    - Predictable computing
    - High network throughput
    - Low network latency
    
- **Profile Three**
  - Workload types
    - Edge functions with very specific computing and networking needs

>     - _Examples: vRAN ( vDU , vCU ) **First Edge use cases** https://github.com/cntt-n/CNTT/blob/master/doc/tech/usecases.md

Reviewer: ```suggestion
    - _Examples: vRAN ( vDU , vCU ) [First Edge use cases](../../tech/usecases.md)
```

# 2.3 Cloud Infrastructure Profiles

Based on the above analysis, following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)

> - **Basic**: For Workloads that can tolerate resource over-subscription and variable latency.

Reviewer 2: Any category you see for compute intensive or special purpose compute like GPUs etc. ?

Author: @msidana , the initial profiles include compute-intensive but we removed it as there is no use cases for it that time

> - **Network Intensive**: For Workloads that require predictable computing performance, high network throughput and low network latency.

Reviewer 2: Do we need to include **predictable computing** performance for network intensive ? This category should stick to networking requirements only. Thoughts ?

Author: @msidana , the initial profiles include compute-intensive but we removed it as there is no use cases for it that time

> - **Edge** : For Edge use cases that requirment sepical requirements, in terms of offloading some functions and with very low network latency.

Reviewer: Will this be exclusive for edge use cases?

Author: @CsatariGergely Yes

> - **Edge** : For Edge use cases that requirment sepical requirements, in terms of offloading some functions and with very low network latency.

Reviewer 2: For Edge use cases that have special~~requirment sepical~~ requirements, in terms of offloading some functions and with very low network latency.

Reviewer 3: "...  and with very low network latency." --> "...  and very low network latency."

> - **Edge** : For Edge use cases that requirment special requirements, in terms of offloading some functions and with very low network latency.

Reviewer 3: suggest
```suggestion
- **Edge** : For Edge use cases that have special requirements, in terms of offloading some functions and with very low network latency.
```

Author: done @pgoyal01 thx

# 2.3 Cloud Infrastructure Profiles

Based on the above analysis, following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
- **Basic**: For Workloads that can tolerate resource over-subscription and variable latency.
- **Network Intensive**: For Workloads that require predictable computing performance, high network throughput and low network latency.

> - **Edge** : For Edge use cases that have special requirements, in terms of offloading some functions , very low network latency ,very high storage requirements and very high computing requirements "GPU" .

Reviewer: Do we really need GPU-s on the edge? For me GPU-s are more for ML training and I do not see an use case for this on the edge.

Reviewer 4: @CsatariGergely Hello Gergely, we have the use case of the remote Edge Survellience with video analytics. Where the live streams need to be analyzed at the Edge (ML-assisted) and alarms are raised for the suspected scenes. It is one of the 5G use cases for Mission Critical IoT.

> - **Edge** : For Edge use cases that have special requirements, in terms of offloading some functions , very low network latency ,very high storage requirements and very high computing requirements "GPU" .

Reviewer: ```suggestion
- **Edge** : For Edge use cases that have special requirements, in terms of offloading some functions, very low network latency, very high storage requirements and very high computing requirements "GPU".
```

Based on the above analysis, following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
- **Basic**: For Workloads that can tolerate resource over-subscription and variable latency.
- **Network Intensive**: For Workloads that require predictable computing performance, high network throughput and low network latency.

> - **Edge** : For Edge use cases that have special requirements, in terms of offloading some functions, very low network latency, very high storage requirements and very high computing requirements "GPU".

Reviewer 3: Does Edge have "very high storage requirements"? Many Edge use cases do not have even medium storage requirements. 
I think "GPU" needs to be deleted.

```suggestion
- **Edge** : For Edge use cases that have special requirements, in terms of offloading some functions, very low network latency and very high computing performance requirements.
```

Author: @pgoyal01 for vRAN use cases there is no need for GPU , but for the edge to make it more wider i belive GPU required for other use cases 

i will copy karim comments 

karim quote
"
Hello Gergely, we have the use case of the remote Edge Survellience with video analytics. Where the live streams need to be analyzed at the Edge (ML-assisted) and alarms are raised for the suspected scenes. It is one of the 5G use cases for Mission Critical IoT."

what do u think

Reviewer 3: @ASawwaf we need to reword the sentence. There is also the issue about the " very high storage requirements" for the Edge. 

```suggestion
- **Edge** : For use cases that have special requirements, in terms of offloading some functions, very low network latency, and very high computing performance requirements. For example, the use of GPU for remote surveillance with video analytics where the live streams are, ML-assisted, analyzed at the Edge and alarms are raised for the suspected scenes. 
```

Reviewer 5: GPUs should be in a different workload profile. One could request that hostgroups supporting the GPU workload profile be deployed in edge datacenters.

Reviewer 5: Similar for storage requirements.  If high storage requirements are needed, they should be identified as a separate workload profile, with a supporting host aggregate, and then that can be deployed at the edge if needed.
